controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/user/Roles.java

```diff
@@ -45,20 +44,17 @@ public static Role toRole(String value) {
     /** Returns the {@link Role} the given tenant, application and role names correspond to. */
     public static Role toRole(TenantName tenant, String roleName) {
         switch (roleName) {
-            case "tenantOwner": return Role.tenantOwner(tenant);
-            case "tenantAdmin": return Role.tenantAdmin(tenant);
-            case "tenantOperator": return Role.tenantOperator(tenant);
+            case "publicReader": return Role.publicReader(tenant);
+            case "publicUser": return Role.publicDeveloper(tenant);
+            case "publicAdministrator": return Role.publicAdministrator(tenant);
```

Comment:
Changing the names of the roles here means all existing role memberships are lost. This is probably OK.

Reply:
Yes, I talked to @smorgrav about this, and we thought it would be ok. _However_ I am not sure if I like the names :)